### DISCUSSION TITLE
Why am I getting Permission Denied when doing cf-remote install --hub hub --bootstrap hub?

### DISCUSSION THREAD

**Discussion by dasilva95Ro** [cfengine/core#5177] [Q&A] [answered]
Hi
I'm applying the page test - https://docs.cfengine.com/docs/3.21/getting-started-installation.html#:~:text=After%20you%20see%20ssh%20working%2C%20save%20the%20host%20in%20cf%2Dremote%20so%20you%20can%20copy%2Dpaste%20our%20later%20commands%3A -
 And when I get to the cf-remote install --hub hub --bootstrap hub item I get the message,
root@192.168.0.80: Permission denied (publickey,password).
 can you help me?

PS: If I try to run the ssh root@IP command, I get "echo Hello".

**Comment by craigcomstock** [comment]
Sorry to hear it isn't working for you. Did you resolve it in the meantime or still stuck here?

I tried on macOS with VirtualBox and had no problems when I followed the instructions [here](https://docs.cfengine.com/docs/3.21/getting-started-installation-local-virtual-machine.html) for creating the virtual machine.

Can you try this command:

```
cf-remote --log-level=debug  install --hub hub --bootstrap hub > log
```

And send the log if the issue is still occurring?

Thanks!

**Comment by craigcomstock** [comment]
@Rod-Santos wondering if you received my reply? Thanks for trying out cf-remote and CFEngine. Let us know how it's going! Cheers! 🎉

**Comment by dasilva95Ro** [comment, marked as answer]
Hello,

I did not actually receive your last reply, but responding to the questions, I was able to perform the configuration and overcome the message "Permission denied (publickey, password)."

I'm using two VMs (AWS) and both on the same IP network and accessed by the same SSH key.
However, using that SSH key inside VM1 to connect to VM2 would cause the aforementioned error. I needed to create a new Key inside VM2 and use this connection file to work the connection between the Hosts.

**Reply by craigcomstock** [reply]
Glad you got it worked out. Sounds like it was an ssh key problem and not related directly to cf-remote. Let us know if you have any other issues or questions! We are happy to help!
